### ISSUE TITLE
Literal value for dct:rights

### ISSUE THREAD

**Issue by tombaker** [dcmi/usage#30]
See [note_rights.md](../blob/master/proposals/2018_iso-related/note_rights.md).

The existing definition and comment for [http://purl.org/dc/terms/rights](http://dublincore.org/documents/2012/06/14/dcmi-terms/#terms-rights) read:

    Information about rights held in and over the resource.
    
    Typically, rights information includes a statement about various property
    rights associated with the resource, including intellectual property
    rights.

The ISO WG proposes to add a second comment:
    
    The Rights element may be used for either a textual statement or a URI
    pointing to a rights statement, or a combination, when a brief statement
    and a more lengthy one are available.

The range of dct:rights is [http://purl.org/dc/terms/RightsStatement](http://dublincore.org/documents/2012/06/14/dcmi-terms/#terms-RightsStatement).

**Comment by aisaac** [comment]
Here's my suggestion for comment:

   Recommended best practice is to use this property with a URI for the rights statement 
   (represented as an instance of dcterms:RightsStatement or another equivalent or more 
   specific class). This URI may be defined by a formal rights statements vocabulary or it 
   may represent an adhoc statement.
   It is acceptable, when this is not possible, to use this property with a literal (name or label) 
   that refers to such a statement, or with the full text of the statement.

Notes: 
- I was less certain about the relevance of "or another equivalent or more specific class" but I left it in for discussion.
- by rights statements vocabulary I mean vocabularies like the set of Creative Commons licenses and (public) marks, or the statements defined at RightsStatements.org. We could add such reference if this is thought to help implementers.

**Comment by osma** [comment]
+1 for the suggested text, and also +1 for mentioning recommended vocabularies.

**Comment by vcharles89** [comment]
Re- the use codes I indeed think we should add a note recommending to not use them.

**Comment by tombaker** [comment]
I almost upvoted this, but agree that it would be good to recommend vocabularies.  A bit more concise, adding references to Creative Commons and RightsStatements.org:

    Recommended practice is to refer to a rights statement with a URI, preferably
    from a formal rights statements vocabulary such as those defined by Creative Commons
    [CC] or RightsStatements.org [RIGHTS].  When no such URI is available, it is
    acceptable to refer to the rights statement with a literal denoting such a
    statement or with the full text of the statement.

**Comment by kcoyle** [comment]
I don't think we should actively discourage "CC-BY"-type statements. They are way better than nothing, and if someone is cataloging from a copy of a resource, that may be all that they have without going through extra effort.

**Comment by osma** [comment]
If we want to allow codes, then we could at least recommend them to be more specific. "CC BY" is somewhat ambiguous, "CC BY 4.0" is somewhat better and "CC BY 4.0 International" actually identifies a particular licence (whose full name is "Creative Commons Attribution 4.0 International").

**Comment by juhahakala** [comment]
We have discussed rights and licenses in metadata a lot here in Finland, although mainly in MARC 21 context. But the same principles may apply in Dublin Core as well. 

Copyright status (rights) and license should be dealt with separately. A resource can be copyrighted or free of copyright. In the former case, the rights holder can allow usage of the document based on a license.  

If there is a license, it should not be recorded in dct:rights, but in dct:license. Rights statements at http://rightsstatements.org/page/1.0/?language=en can be used to describe copyright status.  

Ideally license is widely known and available in the Internet. CC licenses are an example of that. They can and should be recorded as URIs, such as 

https://creativecommons.org/licenses/by/4.0/

for CC BY 4.0 International. One reason for using URI is that license codes and names can be written in many different ways, and although humans can easily parse all possible variants, computers can not. 

From the above page it is possible to access the full text of the license, and large number of translations. But at least in Finland rights holders often create their own license agreements, or use unhelpful statements such as "All rights reserved". Encoding the terms of these occasionally exotic licenses in machine understandable form can be challenging. My favourite (true) example is "This archive material can be used by everyone except journalists from the Finnish Broadcasting Corporation".

**Comment by tombaker** [comment]
The latest iteration of the ISO draft adds a second comment for dct:rights which, when slightly edited to match the DCMIMT style,  reads:

    Note 2 to entry: Recommended practice is to refer to a rights
    statement with a URI.  If this is not possible or feasible,
    a short textual description of rights may be provided.

Do you approve?

**Comment by kcoyle** [comment]
I don't think this is needed here, but this kind of information could go into a Guide.

**Comment by tombaker** [comment]
## APPROVED 
```
Note 2 to entry: Recommended practice is to refer to a rights statement with a
URI.  If this is not possible or feasible, a literal value (name, label, or 
short text) may be provided.
```

**Comment by aisaac** [comment]
I guess the correct link for the resolution is 
https://github.com/dcmi/usage/blob/master/minutes/2018/2018-08-27.dcub-telecon-minutes.md
